datastore/google/cloud/datastore/transaction.py

```diff
@@ -234,3 +244,20 @@ def commit(self):
         finally:
             # Clear our own ID in case this gets accidentally reused.
             self._id = None
+
+    def put(self, entity):
+        """Ensures the transaction is not marked readonly and calls Batch.put
```

Comment:
1. This should not be primary description of what `put()` does. I would suggest "Add an entity to be committed." as the one-liner and then move this "Ensures ..." bit into the long description.
1. "readonly" -> "read only"